### PR TITLE
Upgraded Helm Charts versions. Added ingressClassName to ingresses

### PR DESCRIPTION
**Pull request by jcortejoso**
### Description

Include changes for upgrading nginx and cert-manager charts to latest versions. Updated prometheus image and added `http://local-loki:3100` as a datarsource for the local grafana.
Added a required change in `ingress` resources with the field `ingressClassName` (default to `nginx`).

### Tested

Used for the upgrade of GKE clusters.

### CODE REVIEW

File: packages/helm-charts/grafana/values-clabs.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ datasources:
       name: Prometheus
       type: prometheus
       url: http://prometheus-server.prometheus:9090
+    - access: proxy
+      name: Loki
+      type: loki
+      url: http://local-loki:3100
```
Comment (lvpeschke) [inline packages/helm-charts/grafana/values-clabs.yaml:17]: Is this already set up anywhere?
Wouldn't this additionally require to send the logs to it from promtail?

**Review by lvpeschke — COMMENTED** [review]
Overall looks good.
I'm wondering what's the rationale for setting `"nginx"` as a default value for `ingressClassName` and repeating the value in all the value files?

**Comment by jcortejoso** [comment]
> Overall looks good. I'm wondering what's the rationale for setting `"nginx"` as a default value for `ingressClassName` and repeating the value in all the value files?

So rationale for this is keeping an updated list of all values used in templates in `values.yaml` file, although I agree in that case the `default` function would not be necessary, that would be useful if someone upgrades a helm release with `--reuse-values` or setting a different values file without the reference in the value file.

File: packages/helm-charts/grafana/values-clabs.yaml (hunk 1 above)
Comment (jcortejoso) [reply packages/helm-charts/grafana/values-clabs.yaml:17]: @alvarof2 [confirmed](https://github.com/celo-org/infrastructure/issues/281#issuecomment-1136906720) me that Loki and an additional local promtail were deployed manually to Forno clusters when he reduced the log volume in grafana cloud (most of them were coming from kong/nginx logs in forno).
We can the local-loki datasource in local grafana just to forno clusters, but as it is something that does not hurt (the extra datasource) and loki has been deployed manually to some of the clusters, also I think it makes sense to have it added to all local grafana instances, and install loki using helm wherever it is needed.

Comment (lvpeschke) [reply packages/helm-charts/grafana/values-clabs.yaml:17]: Can you add a comment about it since it's non obvious from reading the config?

**Comment by lvpeschke** [comment]
> > Overall looks good. I'm wondering what's the rationale for setting `"nginx"` as a default value for `ingressClassName` and repeating the value in all the value files?
> 
> So rationale for this is keeping an updated list of all values used in templates in `values.yaml` file, although I agree in that case the `default` function would not be necessary, that would be useful if someone upgrades a helm release with `--reuse-values` or setting a different values file without the reference in the value file.

I don't have a strong opinion, so okay with this way of doing it.

**Review by lvpeschke — APPROVED** [review]

**Review by alvarof2 — APPROVED** [review]

**Pull request merged**